src/app/login/components/feedback-dialog/feedback-dialog.component.ts

```diff
@@ -0,0 +1,58 @@
+import { Component, EventEmitter, OnInit, Output } from '@angular/core';
+import { FormBuilder } from '@angular/forms';
+
+import { FeedbackEvent } from '../../models/events';
+
+@Component({
+  selector: 'app-feedback-dialog',
+  template: `
+      <h2 mat-dialog-title>{{'feedbackcreatedialog.createfeedback' | translate}}</h2>
+      <form mat-dialog-content [formGroup]="feedbackForm" (ngSubmit)="handleSubmit()">
```

Comment:
IMO it would be better to move everything form related to a separate component (smth like FeedbackFormComponent).  That component would then emit feedback event on submission and inside of this component you would call FeedbackService.sendFeedback and display either a success dialog or an error dialog. So:   FeedbackComponent just displays FeedbackDialogComponent; FeedbackDialogComponent takes care of the backend call and displaying success/error dialog and form related stuff is handled in its child FeedbackDialogFormComponent.